DIPs/DIPxxxx.md

```diff
@@ -0,0 +1,233 @@
+# Enhanced Attribute Defaults
```

Comment:
My first impression from this title was  "this looks like a proposal to change more defaults, e.g. `pure` / `@nogc` / `immutable` by default. The abstract is very clear, so it isn't a big problem, but you might consider changing it to e.g. "Per module attribute defaults".

Reply:
I was debating what to name it, it's not the best name, but I like it the way it is. It is a bit more than just per module attributes, as it also has to address the issue of there not being inverse function attributes.